lib/elixir/lib/calendar.ex

```diff
@@ -1255,6 +1482,13 @@ defmodule NaiveDateTime do
     {{year, month, day}, {hour, minute, second}}
   end
 
+  def to_erl(%{calendar: _, year: _, month: _, day: _,
+               hour: _, minute: _, second: _} = naive_datetime) do
+    naive_datetime
+    |> convert(Calendar.ISO)
```

Comment:
And this should likely be `convert!`. We would have caught it if it was a single clause. ;)